Create a cmake machine file for Compute Canada so that
the ./configure.sh script finds all libraries such as
libxc, wannier90 and elpa.

author: Oliver Stueker

diff --git a/cmake/machines/CC.sh b/cmake/machines/CC.sh
new file mode 100644
index 00000000..bd964286
--- /dev/null
+++ b/cmake/machines/CC.sh
@@ -0,0 +1,24 @@
+# Machine file for Compute Canada
+echo "Using special config for Compute Canada-cluster (w/ iomkl toolchain)" 
+
+#Set the compilers to mpiifort, mpiicc, mpiicpc
+export FC="mpifort -r8"
+export CC=mpicc
+export CXX=mpicxx
+
+FLEUR_LIBRARIES="-lxml2;-lelpa_openmp;-lelpa;-lmkl_blacs_intelmpi_lp64;-mt_mpi;-lhdf5_fortran;-lhdf5;-lxcf90;-lwannier "
+
+echo "ELPA is switched on-manually"
+CLI_ELPA_OPENMP=1 
+FLEUR_INCLUDEDIR="$FLEUR_INCLUDEDIR $EBROOTELPA/include/elpa_openmp-$EBVERSIONELPA/modules "
+
+echo "Broken linker can be used"
+CLI_WARN_ONLY=1
+
+#Set environment variables usefull for external dependencies, e.g. ELPA
+export CFLAGS=-mkl
+export CMAKE_Fortran_FLAGS="$CMAKE_Fortran_FLAGS -mkl " 
+export FCFLAGS="-mkl"
+export LIBS="-lmkl -lmkl_blacs_intelmpi_lp64 -lwannier"
+export SCALAPACK_LDFLAGS="-lmkl_blacs_intelmpi_lp64"
+
